### DISCUSSION TITLE
In Supabase Starter template, when I sign in or sign up, I get this error. [urgent]

### DISCUSSION THREAD

**Discussion by 14790897** [supabase/supabase#22292] [Questions] [answered]
After I add try catch in signUp function, I get this error:
```
Error: NEXT_REDIRECT
    at getRedirectError (webpack-internal:///(rsc)/./node_modules/.pnpm/next@14.0.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/client/components/redirect.js:49:19)
    at redirect (webpack-internal:///(rsc)/./node_modules/.pnpm/next@14.0.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/client/components/redirect.js:60:11)
    at $$ACTION_3 (webpack-internal:///(rsc)/./app/login/page.tsx:184:77)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async C:\git-program\ldo_coin\my-app\node_modules\.pnpm\next@14.0.4_react-dom@18.2.0_react@18.2.0\node_modules\next\dist\compiled\next-server\app-page.runtime.dev.js:39:406
    at async t0 (C:\git-program\ldo_coin\my-app\node_modules\
```
In web, it displays
![image](https://github.com/supabase/supabase/assets/121866954/b00b87b6-3089-47d1-80c9-2fe161433c9e)

**Comment by 14790897** [comment]
I use official template : npx create-next-app -e with-supabase and didn't make any change

**Comment by 14790897** [comment]
It can be inferred that there is a problem with the redirect function

**Comment by charislam** [comment]
Did you wrap your `try`/`catch` around the redirect function? Next.js implements redirects by throwing an error, so putting it inside the `try`/`catch` will break the redirect. Try moving it out and see if that fixes it.

**Reply by 14790897** [reply]
Before I add  try catch, this error has already existed

**Comment by 14790897** [comment]
https://github.com/supabase/supabase/assets/121866954/5d976c66-a5c1-4175-bd42-69e2bd973cca

**Comment by 14790897** [comment]
@charislam

**Comment by 14790897** [comment]
This error doesn't appear  in Firefox, but in google chrome

**Comment by charislam** [comment, marked as answer]
Can you try with all your browser extensions deactivated?

I just pulled a fresh Next.js Supabase template and didn't run into any problems on Chrome, so I'm wondering if it's something to do with your browser
